src/test/java/com/minecolonies/test/AbstractTest.java

```diff
@@ -0,0 +1,40 @@
+package com.minecolonies.test;
+
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.util.LanguageHandler;
+import com.minecolonies.util.Log;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Matchers.anyString;
+import static org.powermock.api.mockito.PowerMockito.*;
+
+/**
+ * Abstract test class to abstract away some common uses functionality in Tests
+ */
+@PrepareForTest({ColonyManager.class, LanguageHandler.class, Log.class})
+@PowerMockIgnore("javax.management.*")
```

Comment:
Ignoring logging feels wrong imo and in case of us debugging tests later, enabled logging is a good thing imo.

And PowerMocking all classes that don't work means reducing the thoughts needed for writing tests. With the overall goal to just write stuff that somehow works later on, even if mc is not really testable.